### ISSUE TITLE
Get rpm signature from InstalledRpms parser

### ISSUE THREAD

**Issue by matysek** [RedHatInsights/insights-core#688]
I work on a rule that needs text like 'Key ID 199e2f91fd431d51'. Looking at tests and source code it looks like any of this attributes should work:
```
shared[InstalledRpms].packages['rpm_name'][0]['rhaheader']
shared[InstalledRpms].packages['rpm_name'][0]['dsaheader']
shared[InstalledRpms].packages['rpm_name'][0]['gpgsig-short']
```

However, none of this option works for me. Could be implemented API in InstalledRpms to properly get gpg signature for rpms?

**Comment by xiangce** [comment]
See #665

**Comment by bfahr** [comment]
I believe there's is an additional problem with the parser that might not be fixed by #665.  The current 1.x parser gets input from this populated spec:
```
"/bin/rpm -qa --qf='%{NAME}-%{VERSION}-%{RELEASE}.%{ARCH}\t%{INSTALLTIME:date}\t%{BUILDTIME}\t%{RSAHEADER:pgpsig}\t%{DSAHEADER:pgpsig}\n'"
```
Output data looks like this:
```
'perl-CPANPLUS-0.88-144.el6.x86_64\tWed 09 Aug 2017 06:53:11 PM CDT\t1478187590\tRSA/8, Fri 13 Jan 2017 01:58:29 AM CST, Key ID 199e2f91fd431d51\t(none)'
```
Parsed data looks like this (`__dict__` of `InstalledRpm` object):
```
{'_distribution': 'el6',
 '_release_sep': '144',
 'arch': 'x86_64',
 'buildserver': '(none)',
 'buildtime': '1478187590',
 'epoch': '0',
 'installtime': 'Wed 09 Aug 2017 06:53:11 PM CDT',
 'name': 'perl-CPANPLUS',
 'release': '144.el6',
 'vendor': 'RSA/8, Fri 13 Jan 2017 01:58:29 AM CST, Key ID 199e2f91fd431d51',
 'version': '0.88'}
```

**Comment by xiangce** [comment]
@bfahr - I have renamed the "pgpsig-short" to "pgpsig_short" in both #665 and #666.

**Comment by csams** [comment]
Closing since this appears to be resolved with #665 and #666.
